src/lib/netlist/nl_factory.cpp

```diff
@@ -11,8 +11,7 @@
 #include "nl_setup.h"
 #include "plib/putil.h"
 
-namespace netlist {
-namespace factory {
+namespace netlist::factory {
 
 	// FIXME: this doesn't do anything, check how to remove
```

Comment:
This is another class that’s got an extra level of indentation now.

Reply:
Oh sorry, this one actually isn’t – it’s just GitHub displaying different files with different tab widths.